src/Callout/CalloutMoreFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import React, { ElementType, FC, useState } from 'react';
+import { Button } from '../Button/Button';
+import { Icon } from '../Icon/Icon';
+
+export interface CalloutMoreFooterProps {
+  /** Utilizzarlo in caso si utilizzo di componenti personalizzati */
+  tag?: ElementType;
+  /** Classi aggiuntive da usare per il componente */
+  className?: string;
+  fileUrl?: string;
+}
+
+export const CalloutMoreFooter: FC<CalloutMoreFooterProps> = ({
+  tag = 'div',
+  className,
+  fileUrl = '#',
+  children,
+  ...attributes
+}) => {
+  const Tag = tag;
+  const classes = classNames('collapse-div', className);
+
+  const [isOpen, setOpen] = useState(false);
+
+  return (
+    <Tag {...attributes} className={classes}>
+      <div className='collapse-header' id='heading1'>
```

Comment:
better dynamic or progressive id

Reply:
You are right, the risk is to have more than one element with the same id. But I have to admit I don't really know how to make that progressive or dynamic. The only ways I know are React 18's `useId` hook or lodash `uniqueId` function, but they both are not usable here. Do you have an idea for this case?

Reply:
If I remove the ids wouldn't we have a11y problems?
- BI labels the collapsible area based on the sibling id
- The button should have a `aria-controls` attribute with the id of the collapsible area

Reply:
it will be passed in the development phase by the developer as props of the component. If you're not wrong, the other components work like this. In the future we may open an issue for an upgrade to React 18

Reply:
it will be passed in the development phase by the developer as props of the component. If you're not wrong, the other components work like this. In the future we may open an issue for an upgrade to React 18

Reply:
Developer cannot pass an id directly to those element, but I can add it as a prop and generate the ids based on that. 
Example:
```js
<CalloutMoreFooter id='random-footer'> ... </CalloutMoreFooter>
```

So that `heading1` becomes `random-footer-heading`

I'll soon commit this change

Reply:
ok, can you add a fallback for the id? so as to avoid undefined

Reply:
The fallback would still be static, it wouldn't make much of a difference. BTW I pushed the change but I'm not exactly excited about it 😅